content-services/latest/install/ansible.md

```diff
@@ -0,0 +1,489 @@
+  hosts:
+    repository_1:
+    connection: shh
+    ansible_host: 50.6.51.7
+    ansible_private_key_file: "/path/to/ssh_key.pem"
+    ansible_user: centos
+    ansible_ssh_common_args: "-o UserKnownHostsFile=/dev/null -o ControlMaster=auto
+      -o ControlPersist=60s -o ForwardX11=no -o LogLevel=ERROR
+      -o IdentitiesOnly=yes -o StrictHostKeyChecking=no"
+```
+
+
+## Single Machine Installation
+
+The diagram below shows the result of a single machine installation.
+
+![](../../images/acs-single-machine.png)
+
+Once you have prepared the target host and configured the inventory_ssh.yaml file as described above, you are ready to run the playbook.
+
+To check your inventory file is configured correctly and the control node is able to connect to the target host navigate to the folder you extracted the ZIP to and run the following command:
+
+```bash
+ansible all -m ping -i inventory_ssh.yml
+```
+
+To install ACS 7 Enterprise on the target host execute the playbook as the current user using the following command:
+
+```bash
+ansible-playbook playbooks/acs.yml -i inventory_ssh.yml
+```
+
+Alternatively, to install an ACS 6.2.N Enterprise system use the following command:
+
+```bash
+ansible-playbook playbooks/acs.yml -i inventory_ssh.yml -e "@6.2.N-extra-vars.yml"
+```
+
+> NOTE: The playbook takes around 30 minutes to complete.
+
+Once the playbook is complete Ansible will display a play recap to let you know that everything is done, similar to the block below:
+
+```bash  
+PLAY RECAP *******************************************************************************************************
+activemq_1                 : ok=24   changed=0    unreachable=0    failed=0    skipped=17   rescued=0    ignored=0
+adw_1                      : ok=24   changed=6    unreachable=0    failed=0    skipped=6    rescued=0    ignored=0
+database_1                 : ok=20   changed=0    unreachable=0    failed=0    skipped=11   rescued=0    ignored=0
+nginx_1                    : ok=21   changed=8    unreachable=0    failed=0    skipped=8    rescued=0    ignored=0
+repository_1               : ok=92   changed=43   unreachable=0    failed=0    skipped=14   rescued=0    ignored=0
+search_1                   : ok=34   changed=13   unreachable=0    failed=0    skipped=11   rescued=0    ignored=0
+syncservice_1              : ok=39   changed=18   unreachable=0    failed=0    skipped=13   rescued=0    ignored=0
+transformers_1             : ok=81   changed=10   unreachable=0    failed=0    skipped=44   rescued=0    ignored=0
+```
+
+Once ACS has initialized access the system using the following URLs with a browser:
+
+* Digital Workspace: `http://<target-host-ip>/workspace`
+* Share: `http://<target-host-ip>/share`
+* Repository: `http://<target-host-ip>/alfresco`
+* API Explorer: `http://<target-host-ip>/api-explorer`
+
+## Multi Machine Installation
+
+The diagram below shows the result of a multi machine installation.
+
+![](../../images/acs-multi-machine.png)
```

Comment:
Image is not visible and I think the image link is not specified correctly, should be something like:
![acs-multi-machine]({% link content-services/images/acs-multi-machine.png %})